internal/backend/crypto/gpg/cli/binary_others.go

```diff
@@ -0,0 +1,14 @@
+// +build !windows
+
+package cli
+
+import (
+	"os/exec"
+)
+
+func detectBinary(name string) (string, error) {
+	if name == "" {
+		name = "gpg"
```

Comment:
how about `gpg2` when `gpg` is not found?

Reply:
And shouldn't this `GOPASS_GPG_BINARY` be used here in case it is set??

Reply:
This is what led to this convoluted implementation in the first place.
Are there many distros still shipping gpg 1.x (as `gpg`) and gpg 2.x (as `gpg2`)?